src/sentry/seer/anomaly_detection/types.py

```diff
@@ -45,6 +45,11 @@ class DetectAnomaliesRequest(TypedDict):
     context: AlertInSeer | list[TimeSeriesPoint]
 
 
+class DeleteRuleRequest(TypedDict):
+    organization_id: int
+    alert: dict
```

Comment:
Can we make this match more closely: https://github.com/getsentry/seer/blob/main/src/seer/anomaly_detection/models/external.py#L93-L96
It seems like we're missing the project ID too

Reply:
We can't pass the project ID because a deleted rule has already been unbound from all its projects. Talked to Ram, who said he'll update the API to make project an optional field.